### PR TITLE
apparmor: include apparmor in stage 0, teardown by default

### PR DESCRIPTION
**Pull request by jan--f**
Fixes: bsc#1137182

Signed-off-by: Jan Fajerski <jfajerski@suse.com>

### CODE REVIEW

**Comment by jschmid1** [comment]
@susebot run teuthology

File: srv/salt/ceph/stage/prep/minion/default.sls [hunk 1]
```diff
@@ -6,6 +6,12 @@ crc_method minion:
     - tgt_type: compound
     - sls: ceph.salt.crc.minion
 
+set apparmor:
```
Comment (jschmid1) [inline srv/salt/ceph/stage/prep/minion/default.sls]: That also needs to be added in all the non-default .sls files

Comment (jan--f) [reply srv/salt/ceph/stage/prep/minion/default.sls]: sigh...yes. Reorganized the common parts into a common.sls, that should help in the future.

**Comment by jschmid1** [comment]
@susebot run teuthology

**Comment by jschmid1** [comment]
@jan--f from the logs:

> 2019-06-06T12:35:31.225 INFO:teuthology.orchestra.run.target192168000031.stdout:  - Specified SLS ceph.stage.0.minion.common in saltenv base is not available on the salt master or through a configured fileserver


common is in `srv/salt/ceph/common.sls` while stage.0 expects it in `ceph.stage.0.minion.common`

**Comment by jschmid1** [comment]
> 2019-06-11T14:06:40.772 INFO:teuthology.orchestra.run.target192168000031.stdout:Parsing orchestration ceph.stage.0 steps... fail
2019-06-11T14:06:40.772 INFO:teuthology.orchestra.run.target192168000031.stdout:
2019-06-11T14:06:40.773 INFO:teuthology.orchestra.run.target192168000031.stdout:An error occurred when rendering one of the following states:
2019-06-11T14:06:40.773 INFO:teuthology.orchestra.run.target192168000031.stdout:    - ceph.apparmor (/srv/salt/ceph/apparmor)
2019-06-11T14:06:40.773 INFO:teuthology.orchestra.run.target192168000031.stdout:
2019-06-11T14:06:40.773 INFO:teuthology.orchestra.run.target192168000031.stdout:Error description:
2019-06-11T14:06:40.773 INFO:teuthology.orchestra.run.target192168000031.stdout:  - Rendering SLS 'base:ceph.apparmor.default-teardown' failed: mapping values are not allowed in this context

**Comment by jan--f** [comment]
@susebot run teuthology

**Pull request merged**